Makefile

```diff
@@ -213,7 +213,7 @@ prettier/install:
 
 .PHONY: dockfmt/install
 dockfmt/install:
-	go get -u github.com/jessfraz/dockfmt
+	go get github.com/jessfraz/dockfmt
```

Comment:
I tried go get for dockfmt in my docker environment I strongly recommend download dockfmt using curl from release binary.

Reply:
Okay

Reply:
revised!